src/testRunner/unittests/config/showConfig.ts

```diff
@@ -145,6 +145,26 @@ namespace ts {
                     }
                     break;
                 }
+                case "listOrElement": {

```

Comment:
Please just add `Debug.fail` instead of this.

Reply:
If I place Debug.fail in the "listOrElement" case block, then I get this error 

function` ts.Debug.fail(message?: string | undefined, stackCrawlMark?: AnyFunction | undefined): never
Expected an assignment or function call and instead saw an expression.eslint

Reply:
You need to call `Debug.fail();`